test/Datadog.Trace.DuckTyping.Tests/ReverseProxyTests.cs

```diff
@@ -0,0 +1,162 @@
+#if !NET452
```

Comment:
Is there a test case we can run on `net452` just to make sure the code is exercised?

Reply:
well no in non public methods, but yeah I can create a test case with only public types.

Reply:
Could we also assert known failure modes for `net452` as well?

Reply:
done in 0e8e287

Reply:
Thanks!